### PR TITLE
Revises to specify image file types used for bootable usb drive.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michaelx.vincerra@intel.com>

### CODE REVIEW

**Comment by mvincerx** [comment]
Please note: This PR is in response to Issue 144
https://github.com/clearlinux/clear-linux-documentation/issues/144

FYI: This approach repeats material found here: 
https://clearlinux.org/documentation/clear-linux/get-started/bootable-usb

If this is approved, I will change similar pages for Mac and Windows.

**Comment by mrkz** [comment]
@mvincerx  Thanks for explicitly mentioning which images are usable for the USB stick case.

@rcaballeromx @mltullis could we also please remove the lines 17-18 of this file?
```
 .. include:: ../../guides/maintenance/image-types.rst
:start-after: image-types-content:
```
One of the issues I got while reading this content was about this table, which makes me think that all images listed on the table are available to be used on the USB stick.

**That** table was the initial part of the confusion actually, so having it removed would help a lot in addition to mention only the `installer` and the `live` images.

**Comment by mltullis** [comment]
@mrkz -- Thanks for this suggestion. On Tuesday, 5/22, we have our weekly cross-team meeting to go through pull requests, getting feedback from all stakeholders. We'll discuss our options and will take action that day.

**Comment by mvincerx** [comment]
@tpleavitt Would you mind reviewing this PR? A second set of eyes will help.  I have merge rights now.

File: source/clear-linux/get-started/bootable-usb/bootable-usb-linux.rst [hunk 1]
```diff
@@ -1,21 +1,21 @@
 .. _bootable-usb-linux:
 
-Create a bootable USB drive on Linux\*
+Create a bootable USB drive on Linux
```
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/clear-linux/get-started/bootable-usb/bootable-usb-linux.rst]: Our web team has asked us not to include the asterisk in headers wherever possible. This issue was addressed in our last Tech Docs meeting.  FYI: I asked the same question.

**Comment by mvincerx** [comment]
@tpleavitt Will you please add an AR, next meeting, for me to make these changes on Mac and Windows files?

**Pull request merged**